src/psyclone/domain/lfric/lfric_constants.py

```diff
@@ -273,28 +273,60 @@ def __init__(self):
         # ---------- Infrastructure module maps -------------------------------
 
         # Dictionary allowing us to look-up the name of the Fortran module,
-        # type and proxy-type associated with each LFRic data type.
+        # type and proxy-type associated with each LFRic data structure type.
+        # Data structure type mandates its proxy name, Fortran intrinsic type
+        # of its data and the kind (precision) for the intrinsic type.
         LFRicConstants.DATA_TYPE_MAP = {
-            # 'real'-valued scalar of kind 'r_def' (used for global
+            # 'real'-valued scalar of kind 'r_def' (also used for global
```

Comment:
Actually, it is correct beause the precisions for each data type are specified in the LFRic infrastructure. I left the comments for the defined data types and defined the precision explicitly as it is defined in the LFRic infrastructure.

Reply:
I left the config-file precisions for integer and logical scalars so I removed the comments there.

Reply:
I can understand your argument and I think I agree, but we should either use the config values or they should not exist. It sounds like they should not exist and should instead be hardcoded here. If that is the way we are going, then I'm happy with the change but I think there should then be a new issue added that explains the need to remove the config values (once the other code has been restructured to use lfric constants rather than calling config directly).